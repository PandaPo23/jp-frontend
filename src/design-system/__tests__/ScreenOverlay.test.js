import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import ScreenOverlay from '../ScreenOverlay';

describe('ScreenOverlay component', () => {
  it('should render as ScreenOverlay', () => {
    const wrapper = mount(<ScreenOverlay />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
