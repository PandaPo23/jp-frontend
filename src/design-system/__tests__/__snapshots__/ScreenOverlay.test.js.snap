// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScreenOverlay component should render as ScreenOverlay 1`] = `
.emotion-0 {
  background-color: rgba(1,1,1,.5);
  bottom: 0;
  left: 0;
  position: absolute;
  right: 0;
  top: 0;
  z-index: 1000;
  position: absolute;
}

<ScreenOverlay
  bg="rgba(1,1,1,.5)"
  bottom={0}
  left={0}
  position="absolute"
  right={0}
  top={0}
  zIndex={1000}
>
  <div
    className="emotion-0"
  />
</ScreenOverlay>
`;
